BankSDK/GiniBankSDK/Sources/GiniBankSDK/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyAccessedAPITypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string>CA92.1</string>
```

Comment:
@zladzeyka I've pushed the changes that we discussed and I want to mention that the Payment info was added just in the GiniBankAPILibrary PrivacyInfo file since there we are doing the actual request to send the data to the server. 
Please let me know if my judgment was ok or not.